src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String QUESTION_CAR_NAME = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
+    private static final String QUESTION_TRY_COUNT = "시도할 회수는 몇회인가요?";
+
+    private static final Scanner sc = new Scanner(System.in);
```

Comment:
```suggestion
    private static final Scanner scanner = new Scanner(System.in);
```
마이너하지만 객체지향생활체조 6번 '축약하지 말라'를 참고해보시면 좋겠네요 🙂

[객체지향생활체조](https://williamdurand.fr/2013/06/03/object-calisthenics/)